xtest/tdfs.py

```diff
@@ -110,7 +112,10 @@ def encrypt(
     if attr_values:
         c += [",".join(attr_values)]
     logger.debug(f"enc [{' '.join(c)}]")
-    subprocess.check_call(c)
+    env = dict(os.environ)
+    if fmt == "nano" and use_ecdsa_binding:
+        env |= {"USE_ECDSA_BINDING": "true"}
+    subprocess.check_call(c, env=env)
```

Comment:
should there be a check to set it to false in the env if use_ecdsa_binding is false? im trying to think if there will be issues if we want to run a ecdsa test then a gmac test, i worry the env variable will persist as true

Reply:
casting os.environ to a dictionary copies it, so it should be fine